readme.md

```diff
@@ -231,6 +231,19 @@ var x = require('xastscript')
 console.log(<music />)
 ```
 
+For [TypeScript][], this can be done by setting `"jsx": "react"` and
```

Comment:
Should it be documented why `jsx` is `react`, even though we have nothing to do with react?

Reply:
I added a link to the TypeScript documentation on configuring JSX. I think this should be sufficient.